packages/api/src/jobs/export.ts

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios'
+import jwt from 'jsonwebtoken'
+import { env } from '../env'
+import { findActiveUser } from '../services/user'
+import { logger } from '../utils/logger'
+
+export interface ExportJobData {
+  userId: string
+}
+
+export const EXPORT_JOB_NAME = 'export'
+
+export const exportJob = async (jobData: ExportJobData) => {
```

Comment:
@jacksonh another question is if we should just use async jobs to export the data instead of calling the cloud run service

Reply:
@sywhb maybe we should try to start with an async job and get a baseline of performance in demo. Both our accounts are quite large there so it would give an idea.